### PR TITLE
Move the statements package into the middlewares package

### PR DESCRIPTION
**Pull request by joelmdesouza**
As in the adapaters package we moved the statements into it, I moved the statements concerning the middlewares into it.

@felipeweb @avelino @crgimenes @lerrua @cassiobotaro

### CODE REVIEW

**Review by crgimenes — APPROVED** [review]
LGTM

**Review by felipeweb — APPROVED** [review]

**Comment by joelmdesouza** [comment]
If the merge fails, it is due to the expiration of the JWT token used in the test.

**Review by marioidival — APPROVED** [review]
LGTM

**Comment by felipeweb** [comment]
@joelmdesouza  could you generate a token that never expires for test proposals in http://jwt.io

File: config_test.go [hunk 1]
```diff
@@ -210,11 +210,11 @@ func TestJWTIsRequired(t *testing.T) {
 func TestJWTSignatureOk(t *testing.T) {
 	app = nil
 	MiddlewareStack = nil
-	bearer := "Bearer eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImpvaG4uZG9lQHNvbWV3aGVyZS5jb20iLCJleHAiOjE1MjUzMzk2MTYsImlhdCI6MTUxNzU2MzYxNiwiaXNzIjoicHJpdmF0ZSIsImp0aSI6ImNlZmE3NGZlLTg5NGMtZmY2My1kODE2LTQ2MjBiOGNkOTJlZSIsIm9yZyI6InByaXZhdGUiLCJzdWIiOiJqb2huLmRvZSJ9.zGP1Xths2bK2r9FN0Gv1SzyoisO0dhRwvqrPvunGxUyU5TbkfdnTcQRJNYZzJfGILeQ9r3tbuakWm-NIoDlbbA"
+	bearer := "Bearer eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImpvaG4uZG9lQHNvbWV3aGVyZS5jb20iLCJleHAiOjE1NDI2NzIwMDAsImlhdCI6MTUxNzU2MzYxNiwiaXNzIjoicHJpdmF0ZSIsImp0aSI6ImNlZmE3NGZlLTg5NGMtZmY2My1kODE2LTQ2MjBiOGNkOTJlZSIsIm9yZyI6InByaXZhdGUiLCJzdWIiOiJqb2huLmRvZSJ9.sPzXKw88K7ViYyC1HYHMG-J5HT2SDhNcTJwa-Al21lqEgkN_v8QKAraUfjX51dxGZjMLJ-nCjIK4lxCW_xCTbw"
```
Comment (felipeweb) [inline config_test.go]: use JWT that never expires

**Review by felipeweb — CHANGES_REQUESTED** [review]

File: config_test.go (hunk 1 above)
Comment (joelmdesouza) [reply config_test.go]: Done

**Review by avelino — APPROVED** [review]

**Pull request merged**